form/TimeTextBox.js

```diff
@@ -46,6 +47,18 @@ define([
 		openDropDown: function(/*Function*/ callback){
 			this.inherited(arguments);
 
+                        // Fix #18683
+                        var selectedNode = query(".dijitTimePickerItemSelected", this.dropDown.domNode),
+                            parentNode=this.dropDown.domNode.parentNode;
+                       if (selectedNode[0]) {
+                             // Center the selected node in the client area of the popup.
+                             parentNode.scrollTop=selectedNode[0].offsetTop-parentNode.clientHeight/2;
```

Comment:
Hmm, shouldn't this take into account the height of the selected node?  Otherwise you are centering the top of the selected node rather than centering the center of the selected node.